### PR TITLE
Add Developer Section / Developers Quick Hack Sheet from cwiki.apache.org

### PR DESCRIPTION
**Pull request by bessbd**

### CODE REVIEW

File: dev-docs/DevelopersQuickHackSheet.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Maven2 Issue, start your build with:
+
+
+```
+MAVEN_OPTS="-Xms512m -Xmx1024m -XX:PermSize=256m -XX:MaxPermSize=512m" mvn package -DskipTests
+```
+
+
+Jira:
+[FlUME-1256](https://issues.apache.org/jira/browse/FLUME-1256)
```
Comment (bessbd) [inline dev-docs/DevelopersQuickHackSheet.md:108]: This should be FLUME

File: dev-docs/DevelopersQuickHackSheet.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    -   Check your Diff by clicking "View Diff"
+        -   **if you see some red fields, check your syntax and fix this
+            please**
+    -   review your request
+    -   publish
+
+
+
+-   Post the review link into the Jira
+-   Write Notes into the Jira, regarding your work
```
Comment (bessbd) [inline dev-docs/DevelopersQuickHackSheet.md:89]: Jira -> JIRA?

File: dev-docs/DevelopersQuickHackSheet.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+```
+git stash (will save your branch and reset the working directory)
+```
+
+
+-   Attach the JIRA-ID.patch to the Jira
+
+
+
+-   Open a review request at <https://reviews.apache.org> against
```
Comment (bessbd) [inline dev-docs/DevelopersQuickHackSheet.md:72]: Maybe we should update this for pull requests

**Review by simonati — APPROVED** [review]
LGTM
Once we have the TOC functionality for GitHub+markdown this page can be improved with it.

**Comment by bessbd** [comment]
I'm about to commit this

**Comment by bessbd** [comment]
@satiatcloudera : thank you for the review!

**Pull request closed without merging**